rest/client.py

```diff
@@ -50,11 +51,11 @@ def order_book(self, currency_pair: str, group: int = 1):
 
         return rsp.json()
 
-    def conversion_rate(self):
+    def conversion_rate(self) -> models.ConversionRate:
         """
         Retrieves current BUY/SELL conversion rate for EUR/USD.
         """
         rsp = requests.get(url=self.base_endpoint + '/eur_usd/')
-        
-        return rsp.json()
-    
\ No newline at end of file
+        conversion_rate = models.ConversionRate(rsp)
+
+        return conversion_rate
```

Comment:
Please first merge this PR: https://github.com/digital-alley/bitstamp/pull/1, which introduces a make request function to gather all requests under "one hood". 

In the PR conversion rate is refactored to fit the changes introduced in the PR. This will have to then be rebased and changed.

Reply:
This isn't a requirement really. It's just a debate who's going to have to rebase their PR :)
Will do however, as mentioned PR is really beneficial. ;)